src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -269,7 +269,7 @@ public CommunalPlace getRandomPlace() {
     }
 
     // This method assigns a random number of neighbours to each Household
-    private void assignNeighbours() {
+    void assignNeighbours() {
```

Comment:
Stylistically, I'd rather we explicitly say "public void..." if we need to make this public

Reply:
Ok